ext/standard/basic_functions.stub.php

```diff
@@ -2111,6 +2154,12 @@ function crc32(string $string): int {}
 /** @refcount 1 */
 function crypt(#[\SensitiveParameter] string $string, string $salt): string {}
 
+function crypt_gensalt(?string $salt = null, int $count = 0): ?string {}
+
+function crypt_preferred_method(): ?string {}
+
+function crypt_checksalt(string $salt): int {}
```

Comment:
And here the parameter should probably be called `$settings`. Also affects the C code, which uses `prefix`.